src/main/java/net/datafaker/providers/base/Company.java

```diff
@@ -36,40 +36,27 @@ public String profession() {
     }
 
     public String buzzword() {
-        if (allBuzzwords == null) {
-            try {
-                lock.lock();
-                if (allBuzzwords == null) {
-                    @SuppressWarnings("unchecked")
-                    List<List<String>> buzzwordLists = (List<List<String>>) faker.fakeValuesService().fetchObject("company.buzzwords", faker.getContext());
-                    List<String> buzzwords = new ArrayList<>();
-                    for (List<String> buzzwordList : buzzwordLists) {
-                        buzzwords.addAll(buzzwordList);
-                    }
-                    allBuzzwords = buzzwords;
-                }
-            } finally {
-                lock.unlock();
-            }
-        }
-        return allBuzzwords.get(faker.random().nextInt(allBuzzwords.size()));
+        return faker.options().nextElement(allBuzzwords.get());
+    }
+
+    private List<String> loadBuzzwords() {
+        List<List<String>> buzzwordLists = faker.fakeValuesService().fetchObject("company.buzzwords", faker.getContext());
+        return buzzwordLists.stream().flatMap(Collection::stream).toList();
```

Comment:
Why stream it then map it to a stream then collect it? Why not just:
`buzzwordLists.stream().collect(Collectors.toList())`

Reply:
no, `buzzwordLists.stream().collect(Collectors.toList())` is the same as just `buzzwordLists`.

The point is that `buzzwordLists` is **list of lists**. And we need to **convert it to a list**.

Reply:
Okay, devil in the detail 👿